src/renderer/services/eth2/client/prysm/types.ts

```diff
@@ -46,4 +46,19 @@ export enum PrysmValidatorStatus {
     EXITING = "EXITING",
     SLASHING = "SLASHING",
     EXITED = "EXITED"
-}
\ No newline at end of file
+}
+
+export type ChainHead = {
```

Comment:
PrysmChainHeadResponse*

make sure IEth2Api returns our objects with appropriate types and conversions

Reply:
you mean here: https://github.com/NodeFactoryIo/ChainGuardian/blob/morrigan/bn-syncing-status/src/renderer/services/eth2/client/prysm/beacon.ts#L83 ?

Reply:
yup, ChainHead should be type we use in our app, and PrysmChainHeadResponse is type we get from their api(byte arrays are hex encoded etc)

Reply:
Ok, but in this case they are the same.